starknet_py/net/models/address.py

```diff
@@ -1,13 +1,10 @@
 from typing import Union, Sequence
 
-from starkware.starknet.core.os.contract_address.contract_address import (
-    calculate_contract_address_from_hash,
-)
 from starkware.starknet.services.api.feeder_gateway.feeder_gateway_client import (
     BlockIdentifier,
 )
 
-
+from starknet_py.utils.crypto.facade import calculate_contract_address_from_hash
 from starknet_py.utils.docs import as_our_module
 
```

Comment:
So now we would have 2 functions that do the same. It previously made sense, as `compute_address` was our wrapper for cairo-lang's `calculate_contract_address_from_hash` function, but now that we implement it ourselves, we'd need to keep only one of these.

Reply:
Ok, but `compute_address` is in the public API. So there are two solutions:
- keep it as it is
- move `calculate_contract_addres_from_hash` to the `address.py` directory (and maybe rename it) because users can't import from `facade.py` (circular imports 🙈)

Reply:
I will replace `compute_address` with implementation from the `calculate_contract_addres_from_hash`

Reply:
Ok, check it now

Reply:
I'd prefer the moving solution.